lib/rubocop/cop/lint/percent_string_array.rb

```diff
@@ -53,6 +53,7 @@ def contains_quotes_or_commas?(node)
           end
         end
 
+        # rubocop:disable Performance/HashEachMethods
         def autocorrect(node)
           lambda do |corrector|
             node.values.each do |value|
```

Comment:
@drenmi: Note to self. Make node extensions that respond to `#values` also respond to `#each_value`.